src/main/java/blusunrize/immersiveengineering/common/blocks/metal/ChuteTileEntity.java

```diff
@@ -172,7 +173,10 @@ else if(facing==Direction.EAST)
 				TileEntity inventoryTile;
 				BlockPos invPos = diagonal?getPos().offset(facing): getPos().down();
 				inventoryTile = world.getTileEntity(invPos);
-				if(!world.isRemote&&inventoryTile!=null&&!(inventoryTile instanceof IConveyorTile))
+				if(!world.isRemote&&inventoryTile!=null&&
+						(!(inventoryTile instanceof IConveyorTile)||
+								(((IConveyorTile)inventoryTile).getConveyorSubtype() instanceof BasicConveyor //TODO: change this to IConveyorTile.isCovered() when the API changes
+										&&((BasicConveyor)(((IConveyorTile)inventoryTile).getConveyorSubtype())).isCovered())))
```

Comment:
I would suggest moving this condition (starting at `inventoryTile != null`) into a separate method and structuring it along the lines of `if (a) return true; else if (b) return false; else return c`. IMO the current state is quite unreadable.

Reply:
yea... I was debating that, too, but didn't go for it because it was single use
it was also still quite readable with the API changed and I didn't want to cause a future garbage method :|
give me a bit, I'll have it done